Sources/MapboxMaps/Annotations/ViewAnnotation.swift

```diff
@@ -0,0 +1,206 @@
+import UIKit
+@_implementationOnly import MapboxCommon_Private
+@_implementationOnly import MapboxCoreMaps_Private
+
+public protocol ViewAnnotationManager: AnyObject {
```

Comment:
This needs to be internal so that we can modify it freely. On iOS, we only make protocols public when a feature requires developers to implement the protocol.

Reply:
Will change it, originally I based it on the Android implementation but I agree it's more suitable that way for iOS.

Reply:
I wonder if we need a protocol at all?

Reply:
If the protocol is unused, please remove it. The main use case for protocols like this is to facilitate mocking in the unit tests.